raiden/api/rest.py

```diff
@@ -258,12 +259,13 @@ def open(self, partner_address, token_address, settle_timeout, balance=None):
                 )
             except EthNodeCommunicationError as e:
                 return make_response(str(e), httplib.REQUEST_TIMEOUT)
+            except InsufficientFunds as e:
+                return make_response(str(e), httplib.CONFLICT)
```

Comment:
The RFC says the exact same thing: https://tools.ietf.org/html/rfc7231#section-6.5.2

I have no idea what is better for this